### PR TITLE
Fix nightly build

### PR DESCRIPTION
**Pull request by bjfish**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -118,6 +118,28 @@ jobs:
...
 
+  test-rust-nightly:
+    docker:
+      - image: circleci/rust:latest
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v4-test-cargo-cache-linux-{{ arch }}-{{ checksum "Cargo.lock" }}
```
Comment (syrusakbary) [inline .circleci/config.yml]: You probably want to append nightly here (if not the cache key will be the same as for the stable one, and will force both to be updated)

File: .circleci/config.yml [hunk 2]
```diff
@@ -118,6 +118,28 @@ jobs:
...
+      - run: make test
+      - run: rustup component add rustfmt
+      - run: make lint
+      - save_cache:
+          paths:
+            - /usr/local/cargo/registry
+            - target/debug/.fingerprint
+            - target/debug/build
+            - target/debug/deps
+          key: v4-test-cargo-cache-linux-{{ arch }}-{{ checksum "Cargo.lock" }}
```
Comment (syrusakbary) [inline .circleci/config.yml]: Same

File: .circleci/config.yml [hunk 3]
```diff
@@ -118,6 +118,28 @@ jobs:
...
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v4-test-cargo-cache-linux-{{ arch }}-{{ checksum "Cargo.lock" }}
+      - run: sudo apt-get install -y cmake
+      - run: rustup default nightly
+      - run: make test
+      - run: rustup component add rustfmt
+      - run: make lint
```
Comment (syrusakbary) [inline .circleci/config.yml]: Maybe nightly lint will differ from stable one.
I think we should remove this one (to only test for lint once)

File: .circleci/config.yml (hunk 2 above)
Comment (bjfish) [reply .circleci/config.yml]: Fixed in latest commit.

File: .circleci/config.yml (hunk 3 above)
Comment (bjfish) [reply .circleci/config.yml]: Removed the lint. I agree.

**Comment by losfair** [comment]
Fixes #34

**Pull request merged**